packages/fxa-admin-server/src/scripts/audit-tokens.ts

```diff
@@ -495,6 +510,11 @@ if (require.main === module) {
     })
     .catch((error) => {
       log.error('audit-tokens', { error });
+      Sentry.captureCheckIn({
+        checkInId,
+        monitorSlug: 'audit-tokens',
+        status: 'error',
+      });
```

Comment:
This isn't new code, but random question&hellip; Should the `.catch()` error path set some `exitCode`? Do we want a non-zero exit code for cron jobs?
I see L482 has `process.on('exit', (code) => log.info('exit', { code }))`, and L519 below calls the `process.exit()`, but don't see anywhere that sets a custom `exitCode` in case of failure. 🤷

Reply:
I think it'd be good to set correct exit codes in all our scripts.

Reply:
agreed, we should add correct exit codes on all scripts.  I'm out of my depth on adding it though

Reply:
I'm going to merge this so it gets in the train and we can see what it looks like in sentry